Upgrade_MS_Activator/Upgrade_MS_Activator.xml

```diff
@@ -0,0 +1,49 @@
+<ObjectDefinition>
+  <variables frozen="0">
+    <variable displayName="host" name="params.host" startIncrement="0" type="String" mandatoryArray="false" visible="true" description="IP address of the target host" groupSeparator="" groupDisplayName="" displayOrder="0" increment="0" refServiceURI="" keepOnImport="false" editable="false" onlyDetailView="false" localVarNameMatch="" remoteVarNameMatch="" arrayCanAdd="true" arrayCanRemove="true" arrayCanMove="true" arrayCanEdit="true" displayNameHeader="" fullDisplayName="" isMandatory="false" isUserLocked="false" isGrouped="false" isSearchable="false" isUniqueGlobal="false"/>
```

Comment:
@amitraj83 
can you use either always upper case for first character of variable display name or always lower case ?

Reply:
Done